### PR TITLE
added auth to config.js

### PR DESCRIPTION
**Pull request by jcjiang**
image

### CODE REVIEW

**Comment by jcjiang** [comment]
Pushed to the wrong branch for the auth additions before this commit - let me know if you would like me to rollback the changes and republish through a PR!

File: src/tutorials/secure-your-app.md [hunk 1]
```diff
@@ -76,7 +76,67 @@ Finally, you will want to add authorization configuration to the API. On the rou
...
+app.MapPost("/todos", [Authorize] async (TodoDb db, TodoItem todo) =>
+{
+    await db.Todos.AddAsync(todo);
+    await db.SaveChangesAsync();
+    return Results.Created($"/todo/{todo.Id}", todo);
+});
+
+```
+
+Let's go ahead and start out API. Call this endpoint without providing a valid `access_token` in the `Authorization` header, like:
```
Comment (DamianEdwards) [inline src/tutorials/secure-your-app.md]: > Let's go ahead and start out API.

Typo?

File: src/tutorials/secure-your-app.md [hunk 2]
```diff
@@ -76,7 +76,67 @@ Finally, you will want to add authorization configuration to the API. On the rou
...
+
+```
+curl -X 'POST' \
+  'http://localhost:[YOUR_PORT_NAME]/todos' \
+  -H 'accept: */*' \
+  -H 'Content-Type: application/json' \
+  --header 'Authorization: Bearer [INVALID_TOKEN] \
```
Comment (DamianEdwards) [inline src/tutorials/secure-your-app.md]: Missing closing quote?

File: src/tutorials/secure-your-app.md [hunk 3]
```diff
@@ -76,7 +76,67 @@ Finally, you will want to add authorization configuration to the API. On the rou
...
+{
+  "error": "invalid_token",
+  "error_description": "This request requires a valid JWT access token to be provided"
+}
+```
+
+Now try that same request with a valid token in the `Authorization` header. You should have been able to POST the item successfully and received a `Response body` which includes the item you just added.
+
+```
+[
```
Comment (DamianEdwards) [inline src/tutorials/secure-your-app.md:130]: Does the POST reply come back with 2 items?

Comment (jcjiang) [reply src/tutorials/secure-your-app.md:130]: It does - the first item was added in an earlier tutorial, but I agree that it's confusing to keep here. I'll edit this code sample

**Pull request merged**